docs/guides/frontity-query-options.md

```diff
@@ -0,0 +1,33 @@
+# Frontity Query Options
+
+"Frontity Query Options" are query string parameters added to the URL that start with `frontity_`.
+
+For example:
+
+```
+https://example.com/my_post?frontity_name=Site-name
+```
+
+Any query parameter in the URL that starts with `frontity_` is a "Frontity Query Option".
+
+They can be used to dynamically configure certain options in the Frontity state.
+
+> It is important to note that Frontity Query Options do not form part of the canonical link. This means that whatever is passed in the `frontity_` query parameter will not be matched by handlers or used to fetch data from WordPress. Its purpose is merely to allow dynamic configuration of Frontity.
+
+Query string parameters that start with `frontity_` are reserved to **send special information to Frontity**, i.e. they are not related to the URL that needs to be rendered (e.g. a post or a page), but instead are used to change the Frontity configuration dynamically.
+
+These parameters are removed from `state.frontity.initialLink` and `state.router.link` because they should not affect the content of what needs to be rendered such as a post or a page.
+
+If Frontity Query Option parameters are present in the URL then they are added to `state.frontity.options`, and the properties are then available to any packages that need them.
+
+> Note that the key names of parameters are camelCased when they are added to `state.frontity.options`. So, for example, the `frontity_source_auth` query string param will become `state.frontity.options.sourceAuth` once added.
+
+> Only key names are transformed in this way, values remain unchanged.
+
+Some "Frontity Query Options" are used by core, such as:
```

Comment:
When we say "core" here, wouldn't `@frontity/core` sound better? Should it be hyperlinked to the documentation of the `@frontity/core` package? 

I would also make it clear that at the moment those are the only Frontity Query Options that Frontity itself uses but more might be added in the future. However, the users can also add their own Frontity Query Options if they want to use them in their packages!